traffic_control/clients/python/trafficops/__init__.py

```diff
@@ -14,6 +14,8 @@
 # limitations under the License.
 #
 
+from .__version__ import __version__
+
 # Import Local Modules into the 'trafficops' package namespace for convenience.
 from .tosession import TOSession
 from .restapi import InvalidJSONError, LoginError, OperationError, api_request, RestApiSession
```

Comment:
what might be better is removing these bits. If you do that, it shouldn't require anything to be imported to get to the version, which stays in one place at the package level. The cost is that you can't really get much else into the current namespace when you do `from trafficops import *` which is honestly something you shouldn't do anyway, because it clobbers.

Reply:
to be clear, by "these bits" I mean the local module imports (and consequently the `__all__` definition). Failing that, the package-level `__version__` should still be defined to be the package's version. Otherwise to get the version from within Python you'd need to do `from trafficops.__version__ import __version__` instead of just `from trafficops import __version__`

Reply:
I'm not totally following the advantages you're proposing. I can get the module version the same way as it was before: 
```
Python 3.6.7 (default, Jan  4 2019, 09:57:41)
[GCC 4.2.1 Compatible Apple LLVM 9.0.0 (clang-900.0.39.2)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import trafficops
>>> trafficops.__version__
'1.0.0'
>>>
```
as well as:
```
Python 3.6.7 (default, Jan  4 2019, 09:57:41)
[GCC 4.2.1 Compatible Apple LLVM 9.0.0 (clang-900.0.39.2)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> from trafficops import __version__
>>> __version__
'1.0.0'
>>>
```

Reply:
I missed this line: `from .__version__ import __version__`. I suppose this is fine, though I was really excited to find a reason to specifically disallow `from trafficops import *`... oh well.

Reply:
Yeah, unfortunately, I think something like that would probably require a deprecation cycle as well so that existing instances of `from trafficops import *` don't break for people without notice.